src/Testing/InteractsWithMail.php

```diff
@@ -147,4 +147,9 @@ protected function seeMessage()
 
         return $this;
     }
+
+    protected function dontSeeMessage()
+    {
+        $this->assertTrue($this->getMessages()->isEmpty());
```

Comment:
Although you wouldn't normally chain it with other mailthief-specific methods, you probably would with any other assertions beyond the scope of this trait, so the fluent interface is to be expected. In my opinion you should definitely be able to do this, for instance, in some integration test:

```
$this->dontSeeMessage()
    ->assertResponseStatus(200);
```